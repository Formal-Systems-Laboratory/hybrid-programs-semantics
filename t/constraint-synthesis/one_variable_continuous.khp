vars a, t ;  t := 0.0 ; a' = 2.0 , t' = 1.0 & a < 10.0 , t < 5.0
