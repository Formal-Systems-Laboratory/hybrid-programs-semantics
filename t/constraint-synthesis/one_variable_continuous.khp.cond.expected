a == 2.*(t_post) + (VAR_a) && t == 0. + 1.*(t_post) && (a_i) == (VAR_a) && ((-10. + 1.*(VAR_a) < 0 && -10. + 2.*(t_post) + 1.*(VAR_a) < 0) || ((t_post) < 0 && -5. + 1.*(t_post) + 0.5*(VAR_a) < 0) || (10. - 1.*(VAR_a) < 0 && -(VAR_a) < 0 && (t_post) < 0)) && (t_i) == (VAR_t) && -5. + 1.*(t_post) < 0 && (t_post) >= 0.
